settings/serverSettings.ts

```diff
@@ -42,27 +42,26 @@ export const BLOG_POSTS_PER_PAGE: number =
     parseIntOrUndefined(serverSettings.BLOG_POSTS_PER_PAGE) ?? 21
 export const BLOG_SLUG: string = serverSettings.BLOG_SLUG ?? "blog"
 
-export const DB_NAME: string = serverSettings.DB_NAME ?? "owid"
-export const DB_USER: string = serverSettings.DB_USER ?? "root"
-export const DB_PASS: string = serverSettings.DB_PASS ?? ""
-export const DB_HOST: string = serverSettings.DB_HOST ?? "localhost"
-export const DB_PORT: number =
-    parseIntOrUndefined(serverSettings.DB_PORT) ?? 3306
+export const GRAPHER_DB_NAME: string = serverSettings.GRAPHER_DB_NAME ?? "owid"
+export const GRAPHER_DB_USER: string = serverSettings.GRAPHER_DB_USER ?? "root"
+export const GRAPHER_DB_PASS: string = serverSettings.GRAPHER_DB_PASS ?? ""
+export const GRAPHER_DB_HOST: string =
+    serverSettings.GRAPHER_DB_HOST ?? "localhost"
+export const GRAPHER_DB_PORT: number =
+    parseIntOrUndefined(serverSettings.DB_PORT) ?? 3307
```

Comment:
Maybe for the default here we should try to stay with 3306? For the blessed setting we override this anyhow but I'm not sure if some dev using local mysql somewhere might not have set this and then it's nicer to stay with the upstream default